pyproject.toml

```diff
@@ -29,24 +29,8 @@ pyyaml = "^6.0.1"
 fire = ">=0.6,<0.8"
 # used for template capabilities
 jinja2 = "^3.1.4"
-# used for report generation capabilities
-plotly = "^5.22.0"
-# used for plotly dependencies
-kaleido = "0.2.1"
-# used for environment detection
-ipython = "^8.12.3"
-# used for image processing
-scikit-image = [
-  { version = "^0.19.3", python = "<3.9" },
-  { version = ">0.19.3", python = ">=3.9" },
-]
-# dependency of scikit-image
-pywavelets = [
-  { version = "^1.4.1", python = "<3.9" },
-  { version = ">1.4.1", python = ">=3.9" },
-]
-# used for image modifications in cytodataframe
-opencv-python = "^4.10.0.84"
```

Comment:
Are these removed because they only applied to CytoDataFrame?

Reply:
Thanks for asking - yes, these were dropped as I found they were only required by CytoDataFrame.